packages/storage/src/reference.ts

```diff
@@ -236,7 +236,7 @@ export function getStreamInternal(
 
   ref.storage
     .makeRequestWithTokens(requestInfo, newStreamConnection)
-    .then(stream => stream.pipe(result))
+    .then(stream => (stream as NodeJS.ReadableStream).pipe(result))
```

Comment:
Changes in this file seem unrelated?

Reply:
This is "also fix some type errors in storage", storage was spitting out some errors on build that I was concerned about.